assets/stylesheets/_sections.scss

```diff
@@ -110,61 +140,160 @@
+  font-size: $type--medium;
   color: $white;
+  @media screen and (min-width: $bp-small+100){
+    font-size: $type--large;
+  }
+}
+
+
+.blink{
+  color:$lemon;
+  -webkit-animation: blink linear 1.25s infinite;
+  animation: blink linear 1.25s infinite;
+}
+@-webkit-keyframes blink {
+  0% { opacity: 1; } 
+  50% { opacity: .1; } 
+  100% { opacity: 1; }
+}
+@keyframes blink {
+  0% { opacity: 1; } 
+  50% { opacity: .1; } 
+  100% { opacity: 1; }
 }
 
 .arcade-section__teaser {
   font-family: $font-stack--retro;
   font-size: $type--small;
   color: $white;
+  text-align: center;
 }
 
-.events-section {
-  align-items: center;
-  display: flex;
+.events-section{
+  background-color: #020710;
+  overflow: hidden;
 }
 
+
 .events-section__costume,
 .events-section__food {
   position: relative;
 }
-.events-section__image,
-.donations-section__image {
-  max-width: 360px;
+
+.events-section__food {
+  background: 
+    black
+    url("#{$asset-path}" + "#{$food-background}");
+  background-position: center bottom;
+  background-repeat: no-repeat;
+  background-size: auto 100%;
+  width: 320px;
+  height: 320px;
+  @media screen and (min-width: $bp-small+50){
+    width: 400px;
+    height: 400px;
+  }
+  @media screen and (min-width: $bp-large+20){
+    width: 500px;
+    height: 500px;
+  }
+
+}
+.events-section__costume {
+  background: 
+    black
+    url("#{$asset-path}" + "#{$costume-background}");
+  background-position: center bottom;
+  background-repeat: no-repeat;
+  background-size: auto 100%;
+  width: 320px;
+  height: 320px;
+  @media screen and (min-width: $bp-small+50){
+    width: 400px;
+    height: 400px;
+  }
+  @media screen and (min-width: $bp-large+20){
+    width: 500px;
+    height: 500px;
+  }
 }
 
 .events-section__caption {
   position: absolute;
   bottom: 0;
   color: $white;
-  padding: 1em;
+  padding: 3em;
+  width: 100%;
   text-align: center;
   background-image: linear-gradient(-180deg, rgba(2,7,16,0.00) 19%, rgba(2,7,16,0.98) 71%, #020710 100%);
 }
+.events-section__caption p {
+  padding-bottom: 1.5em;
+}
 
-.donations-section  {
-  .wrapper {
-  align-items: center;
-  display: flex;
+.donations-section__image {
+  max-width: 500px;
+  width: 100%;
+  box-shadow: 0 0 40px 0 rgba(0,0,0,0.25);
+}
+
+.donations-section__blurb {
+  
+  @media screen and (min-width: $bp-small){
+    padding-left:3em;
   }
 }
 
 .adios-section {
+  padding-top: 3em;
     background:
         linear-gradient(rgba(black, 0.5), rgba(black, 0.5)),
         black
         url("#{$asset-path}" + "#{$footer-background}");
   background-position: center bottom;
   background-repeat: no-repeat;
   background-size: auto 150%;
+  @media screen and (min-width: $bp-small){
+    padding-top: 6em;
+  }
 }
 
 
 .adios-section__blurb{
   color: $ruby;
+
+  button {
+    margin-top: 1.5em;
+  }
 }
 
 .adios-section__headline {
   font-size: $type--xl;
 }
+
+.logo--small {
+  width: 100%;
+  max-width: 200px;
+}
+
+.upside-up {
```

Comment:
hahah, only on a Stranger Things project would this name make sense

Reply:
lol